node/src/actors/chain_manager/mod.rs

```diff
@@ -1837,7 +1793,10 @@ impl ChainManager {
 
             actix::fut::ok(())
         })
-        .map_err(|e, _, _| log::error!("{:?}", e));
+        .map_err(|(), _, _| {
+            // Errors at this point should be impossible because we explicitly ignore them
```

Comment:
if it is impossible, should we put a panic?

Reply:
I think we can also remove that `map_err`, but I will put a panic so we can find any potential errors.